var/spack/repos/builtin/packages/cmake/package.py

```diff
@@ -324,7 +316,7 @@ def bootstrap_args(self):
         rpaths = spack.build_environment.get_rpaths(self)
```

Comment:
Just above htis line, not currently part of the patch, (it won't let me comment on something not part of the diff):
```
        if '+ownlibs' in spec:
            args.append('-DCMAKE_USE_OPENSSL=%s' % str('+openssl' in spec))
```
It's no longer necessary to gate this with `if '+ownlibs' in spec:` since that's now handled by the `when` clause in the `openssl` variant.